local-libs/vcluster/vclusterops/create_db.go

```diff
@@ -46,25 +45,16 @@ const (
 // Below constants are the key name used to set Vertica TLS configuration from K8S operator
 /* example: {"Namespace": "default", "SecretManager": "kubernetes", "SecretName":"Secret"}*/
 const (
-	TLSSecretManagerKeyNamespace          string = "Namespace"
-	TLSSecretManagerKeySecretManager      string = "SecretManager"
-	TLSSecretManagerKeySecretName         string = "SecretName"
-	TLSSecretManagerKeyCACertDataKey      string = "CADataKey"
-	TLSSecretManagerKeyCertDataKey        string = "CertDataKey"
-	TLSSecretManagerKeyKeyDataKey         string = "KeyDataKey"
-	TLSSecretManagerKeyTLSMode            string = "TLSMode"
-	TLSSecretManagerKeyAWSRegion          string = "AWSRegion"
-	TLSSecretManagerKeyAWSSecretVersionID string = "AWSVersion"
+	tlsSecretManagerKeyNamespace     string = "Namespace"
+	tlsSecretManagerKeySecretManager string = "SecretManager"
+	tlsSecretManagerKeySecretName    string = "SecretName"
+	tlsSecretManagerKeyCACertDataKey string = "CADataKey"
+	tlsSecretManagerKeyCertDataKey   string = "CertDataKey"
+	tlsSecretManagerKeyKeyDataKey    string = "KeyDataKey"
+	tlsSecretManagerKeyTLSMode       string = "TLSMode"
```

Comment:
This overrides my changes in main. Let's merge the PR created after merge to master in gitlab